analyzers/src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicSyntaxHelper.cs

```diff
@@ -1,6 +1,6 @@
 ﻿/*
  * SonarAnalyzer for .NET
- * Copyright (C) 2015-2022 SonarSource SA
+ * Copyright (C) 2015-2023 SonarSource SA
```

Comment:
This should eventually go away after another rebase later. We'll bump headers this week

Reply:
See #6665

Reply:
..merged, you can rebase to deal with these

Reply:
Done